src/screens/BallotEjectScreen.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+
+import { ReviewBallot } from '../config/types'
+import { fetchNextBallotToReview } from '../api/hmpb'
+
+import Main, { MainChild } from '../components/Main'
+import Prose from '../components/Prose'
+import Button from '../components/Button'
+import MainNav from '../components/MainNav'
+
+const Columns = styled.div`
+  display: flex;
+  > div {
+    margin-right: 1em;
+    &:first-child {
+      flex: 1;
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+    img {
+      max-width: 100%;
+      height: 83vh;
+    }
+  }
+`
+
+const RectoVerso = styled.div`
+  display: flex;
+  & > * {
+    &:first-child {
+      margin-right: 1em;
+    }
+  }
+  img {
+    max-width: 100%;
+    height: 83vh;
+  }
+`
+
+interface Props {
+  continueScanning: () => void
+}
+
+const BallotEjectScreen = ({ continueScanning }: Props) => {
+  const [ballotInfo, setBallotInfo] = useState<ReviewBallot | undefined>()
+
+  useEffect(() => {
+    ;(async () => {
+      setBallotInfo(await fetchNextBallotToReview())
+    })()
+  }, [setBallotInfo])
+
+  if (!ballotInfo) {
+    return null
+  }
+
+  return (
+    <React.Fragment>
+      <MainNav>
+        <Button small onPress={continueScanning}>
+          Continue Scanning Batch
+        </Button>
+      </MainNav>
+      <Main>
+        <MainChild maxWidth={false}>
+          <Columns>
+            <Prose maxWidth={false}>
+              <h1>Remove This Ballot</h1>
+              <p>Human review is required for this last scanned ballot.</p>
+              <p>
+                Once the ballot has been removed, press the button to continue
+                scanning.
+              </p>
+            </Prose>
+            <RectoVerso>
+              <Prose>
+                <h4>Front</h4>
+                <p>
+                  <img
+                    src={`/scan/hmpb/ballot/${ballotInfo.ballot.id}/front/original`}
+                    alt="p1"
+                  />
+                </p>
+              </Prose>
+              <Prose>
+                <h4>Back</h4>
+                <p>
+                  <img
+                    src={`/scan/hmpb/ballot/${ballotInfo.ballot.id}/back/original`}
```

Comment:
I'd really prefer to use the URL embedded in the response if possible. Also, we almost certainly do not want original, but normalized. Otherwise you end up with upside-down images.